### PR TITLE
Configuring Codecov

### PR DESCRIPTION
**Pull request by zoewangg**
## Description
- Ignore test files for code coverage

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ignore:
+  - "codegen-lite/test/resources/**/*"
+  - "codegen/src/test/resources/**/*"
+  - "test/**/*"
```
Comment (varunnvs92) [inline .codecov.yml]: Extra character at the end

Comment (zoewangg) [reply .codecov.yml]: Hmm, not sure whey it showed up like that, but there's no extra character in this file though

Comment (varunnvs92) [reply .codecov.yml]: Any reason why we are just ignoring these folders? Are there others that need to be excluded?

Comment (zoewangg) [reply .codecov.yml]: This is what we discussed this morning, excluding the test folders for the test coverage. Double checked and those are the only folders we want to ignore.

Comment (zoewangg) [reply .codecov.yml]: Oh, I misunderstood your question. Yeah, those folders are the ones showing on codecov page that should not be counted.

screen shot 2019-03-01 at 3 36 31 pm

**Pull request merged**